src/extras.jl

```diff
@@ -69,8 +52,8 @@ also accept them.
   the intervals; or a function `f(from, to, i; leftclosed, rightclosed)` that generates
   the labels from the left and right interval boundaries and the group index. Defaults to
   `"[from, to)"` (or `"[from, to]"` for the rightmost interval if `extend == true`).
-* `allowmissing::Bool=true`: when `true`, values outside of breaks result in missing values.
-  only supported when `x` accepts missing values.
+* `allowoutside::Bool=false`: when `false`, an error is thrown if values outside of breaks
```

Comment:
actually looking at `extend` and `allowoutside` they are duplicating functionality. Maybe `extend` should take three values `true` and `false` would work as they do now and if you pass `missing` then you get the behavior where `missing` is returned.

Reply:
Yes, good idea. I've just pushed a commit to do that. It's a bit weird to use `extend=missing` to set values outside breaks to `missing`, but I couldn't find a better solution. We could have `outside=:error`, `outside=:extend` and `outside=missing`, but that's not super nice either. Ideas welcome...

Reply:
OK - let us wait what other think, my reading of `extend=missing` is "extend with `missing`"